themes/buildpacks/layouts/shortcodes/in-depth-comparison-table.html

```diff
@@ -453,6 +457,44 @@
             </div>
           </td>
         </tr>
+
+        <tr>
+          <td class="border-left border-right">
+            <div style="overflow-x: auto; text-align: left">
+              <p class="text-pink font-weight-bold">Governance</p>
+              CNCF
+            </div>
+          </td>
+          <td class="comp-dockerfiles">
+            <div style="text-align: left">
+              <p></p>
+              <br />
```

Comment:
Why do we need this empty paragraph (`<p>`) and  break (`<br/>`)?

Reply:
That was for the layout, to align the start of text with the left column, i'll make this cleaner with CSS :)